docs/code_examples/aec_rps.py

```diff
@@ -60,29 +60,36 @@ def __init__(self, render_mode=None):
         The init method takes in environment arguments and
          should define the following attributes:
         - possible_agents
-        - action_spaces
-        - observation_spaces
+        - render_mode
+
+        Note: the action_spaces and observation_spaces attributes are deprecated.
+        Spaces must be defined in the action_space() and observation_space() methods.
```

Comment:
I think a better wording would be:
`As of vX.X.X, action and observation spaces are to be defined in the action_space and observation_space methods`.